src/2sv/smartphone/DownloadApp.vue

```diff
@@ -19,10 +19,10 @@
       </figure>
 
       <figure class="badges d-flex align-center justify-center">
-        <a :href="`https://play.google.com/store/apps/details?id=${playStoreId}`" target="_blank" class="pr-4">
+        <a :href="playStoreUrl" target="_blank" class="pr-4">
```

Comment:
Seems like this could still be flagged by Sonar. Since we know it's not a user-supplied URL, maybe it's good enough just to mark it as safe in the Sonar console.

Reply:
I was just curious if it was the string template that was causing the warning and others like it. Sonar didn't fail, so does that mean they don't test the entire repo every time?